application/src/main/resources/ecc.yml

```diff
@@ -84,6 +84,13 @@ repair:
   interval:
     time: 7
     unit: days
+
+  ##
+  ## How long from current time should the last repaired time be simulated/assumed for new tables.
```

Comment:
This is not quite right.

I suggest some more of:

Initial delay for new tables. New tables are always assumed to have been repaired in the past, set off by the
repair interval. This will result in an immediate first repair. However, with this setting an initial delay can
be set to back off the first repair. Defaults to one day.